### ISSUE TITLE
Submission form: add option to zip files before upload

### ISSUE THREAD

**Issue by Maurier** [cznethub/dsp#95]
**Describe the feature you'd like and what it will do**
Option to zip the selected files and upload as such.

**Additional context**
Suggested during our team meeting on Feb 8th, 2022.

**Comment by horsburgh** [comment]
@Maurier - Maybe I'm not understanding this. Can't a user just zip the files on their local computer and then upload the zip file? Why does the DSP need to manage the zipping?

**Comment by Maurier** [comment]
Yes, users can do that. This was suggested during one of our weekly calls. I suppose it would be a quality of life improvement (and perhaps we don't need it).
